share/lib/hoc/import3d/import3d_sec.hoc

```diff
@@ -197,7 +197,7 @@ obfunc contourcenter() {local i localobj mean, pts, perim, d
 	perim = new Vector(pts.ncol)
 	for i=1, pts.ncol-1 { perim.x[i] = perim.x[i-1] + pts.getcol(i-1).mag }
 	d = new Vector(101)
-	d.indgen(perim.x(perim.size-1)/100)
+	d.indgen(perim.x[perim.size-1]/100)
```

Comment:
Is this change still needed? I understood that later versions of the PR are more permissive?